frontend/storybook/src/components/entry/JIG/JIG-panel.ts

```diff
@@ -0,0 +1,145 @@
+ import "@elements/entry/jig/publish/panel/jiggling-body";
+import "@elements/entry/jig/publish/panel/side-bar-number";
+import "@elements/entry/home/TOSORT/column-list";
```

Comment:
why are we importing stuff from the homepage into jig? if it's generic enough to be used everywhere - move it into `core`. we have a `lists` folder...